### PR TITLE
Add "Get Started" modal to profile pages with bootstrap

### PR DESCRIPTION
**Pull request by winniewan**
- This "Get Started" modal gives user the context and support for using their profile on our platform. 
- Only users that are logged in and are viewing their own profile gets to see this feature.
- User can click anywhere on the screen or "Close" button to exit the modal.

### CODE REVIEW

File: capstone/src/main/webapp/business.html [hunk 1]
```diff
@@ -223,5 +224,71 @@ <h5><label for="support">How can we support you?</label></h5>
...
+                    <h5>Welcome to our platform for supporting local businesses!</h5>
+                    <p>This guide will help you get started with using our platform. First, let's build your profile!</p>
+                  </div>
+
+                  <div>
+                    <h5>Where to edit your profile?</h5>
+                    <p>
+                      Across from the "Get Started" button, there's an "Edit Profile" button.<br> 
+                      Click on that to get started in creating your profile.<br>
+                      There are two main users: Business owners and Local people.<br>
```
Comment (eash130) [inline capstone/src/main/webapp/business.html:250]: suggested change:
Business -> business
Local people -> general users

Comment (winniewan) [reply capstone/src/main/webapp/business.html]: yep, sure!

File: capstone/src/main/webapp/business.html [hunk 2]
```diff
@@ -223,5 +224,71 @@ <h5><label for="support">How can we support you?</label></h5>
...
+                      edit your profile anytime! 
+                    </p>
+                    <p>
+                      Now you can explore the platform! Feel free to checkout other businesses, follow them, post comments, share their profile, and support one another during this pandemic time. 
+                    </p>
+                    <p>
+                      Thank you for helping your local businesses! 
+                    </p>
+                    <p>
+                      If there's any questions, feel free to find support at this email.
```
Comment (alygerharter) [inline capstone/src/main/webapp/business.html:280]: delete this sentence if you don't have a support email, especially if you aren't planning on running one :)

File: capstone/src/main/webapp/profile.html [hunk 3]
```diff
@@ -172,5 +173,71 @@ <h5><label for="support">How can we support you?</label></h5>
...
+                      edit your profile anytime! 
+                    </p>
+                    <p>
+                      Now you can explore the platform! Feel free to checkout other businesses, follow them, post comments, share their profile, and support one another during this pandemic time. 
+                    </p>
+                    <p>
+                      Thank you for helping your local businesses! 
+                    </p>
+                    <p>
+                      If there's any questions, feel free to find support at this email.
```
Comment (alygerharter) [inline capstone/src/main/webapp/profile.html:229]: delete

File: capstone/src/main/webapp/business.html (hunk 2 above)
Comment (winniewan) [reply capstone/src/main/webapp/business.html]: got it! done.

File: capstone/src/main/webapp/profile.html (hunk 3 above)
Comment (winniewan) [reply capstone/src/main/webapp/profile.html]: done

**Pull request merged**